### PR TITLE
Linear logic

### PR DESCRIPTION
**Pull request by willyd332**
Did some research on the math involved. Much easier than it seems (because I'm creating the board myself), so most of the readMe is irrelevant yet interesting imo. 
On test is failing, even though the thing it's testing (the winModal) it clearly working when the app runs.

Also some styling

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,3 +6,161 @@
...
+- The order in which the tiles are pressed is irrelevant to the final result
+- Pressing the same tiles used in the creation of a board will result in a return to the original state
+- Therefore, by finding the tiles pressed to acheive any given state (if they exist), we have found a solution to the board
+
+### Calculations
+##### *All Calculations Will Be In **mod(2)***
+`Let b = [b1, b2, b3, ..., b25]`
+where `bi` represents the state of tile `i`, `0 = *black* and 1 = *white*`
+
+`Let x = [x1, x2, bx, ..., x25]`
```
Comment (averyvery) [inline README.md:34]: Still trying to follow this, haven't done this kind of math before, but should this be `x3`, not `bx`?

File: README.md [hunk 2]
```diff
@@ -6,3 +6,161 @@
...
+
+Thus `b` is solvable if it is orthogonal to these two vectors.
+
+##### Solving A Board
+
+We can see that:
+`[e1, e2, e3, ..., e25] = e24*n1 + e25*n2`
+
+Normally, becasue we have paremeters, `A` would be inconsistent and have infinite solutions,
+hovever, because we are in modulo 2, we can limit this to four potential solutions.
```
Comment (averyvery) [inline README.md:146]: Supe minor note, but might want to run spell-check on this — I think there are a few typos (hovever, becasue, iff).

Comment (willyd332) [reply README.md:146]: iff = if and only if

but the others are typos

File: README.md (hunk 1 above)
Comment (willyd332) [reply README.md:34]: Good call

File: README.md (hunk 2 above)
Comment (averyvery) [reply README.md:146]: Ahhh, TIL.

**Pull request merged**